build/source/engine/getVectorz.f90

```diff
@@ -528,6 +559,9 @@ subroutine varExtract(&
   end do  ! looping through non-missing energy state variables in the snow+soil domain
  endif
 
+ ! extract temperature of the canopy air space
```

Comment:
Update comment - I don't think you're extracting the `temperature of the canopy air space` here